howso/client/pandas/client.py

```diff
@@ -221,40 +222,48 @@ def react_series(
 
         Returns
         -------
-        dict
-            A dictionary with keys `series` and `explanation`. Where `series`
-            is a DataFrame of feature columns with series values as rows,
-            and `explanation` is a dict with the requested audit data.
+        Reaction:
+            A MutableMapping (dict) with these keys -> values:
+                actions -> pandas.DataFrame
+                    A data frame with columns specified in
+                    ``features_to_generate`` list (or list of lists).
```

Comment:
I may be delusional... but I'm pretty sure that the columns in this returned dataframe will be determined by 'action_features'. 

'features_to_generate' is the Synthesizer parameter I think?